codes/data/france/format_infected.do

```diff
@@ -6,7 +6,7 @@
 // Please input last date info
 local y = 2020 
 local m = 3 
-local d = 18
+local d = 25
```

Comment:
We are dropping the cutoff date from data collection scripts, and doing the cutoffs only at the regression stage. Can we remove this functionality from `format_infected.do`?

Reply:
@sannanphan if we're no longer using this section can we remove it? I think confusing for people to see it if changing the date no longer has an effect. If there's a reason for it to stay in comments, just resolve this convo